app/containers/ActionSheet/ActionSheetContentWithInputAndSubmit/index.tsx

```diff
@@ -102,6 +103,7 @@ const ActionSheetContentWithInputAndSubmit = ({
 	showInput?: boolean;
 	inputs?: { placeholder: string; secureTextEntry?: boolean; key: string }[];
 	isDisabled?: (inputValues: string[]) => boolean;
+	autoComplete?: TextInputProps['autoComplete'];
```

Comment:
Didn't get it.
You did it on ActionSheetContentWithInputAndSubmit, but I search for `<ActionSheetContentWithInputAndSubmit` and there's no `autoComplete` prop added anywhere.
Is it done on another PR?